apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/database/model/DbPerson.java

```diff
@@ -83,17 +83,4 @@ public List<DbAlias> getAlias() {
         }
         return alias;
     }
-
-    @Override
```

Comment:
Endringen din betyr at equals og hash tar hensyn til alle felter og ikke kun id, er det ønskelig?

hashCode under bør inkludere id også for å samsvare med equals (?)